Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -587,7 +587,7 @@ namespace xr
             anchorSpaceCreateInfo.poseInAnchorSpace = xr::math::Pose::Identity();
             CHECK_XRCMD(apiExtensions.xrCreateSpatialAnchorSpaceMSFT(session, &anchorSpaceCreateInfo, anchorSpace->Space.Put()));
 
-            const auto nativeAnchorPtr = reinterpret_cast<NativeAnchorPtr>(anchorSpace.get());
+            const auto nativeAnchorPtr = reinterpret_cast<NativeAnchorPtr>(anchorSpace->Anchor.Get());
```

Comment:
@satyapoojasama  and @rgerd  we need to confirm that we should expose the anchor compared to a space here.

Reply:
This looks correct to me. There's currently a 1-1-1 mapping of Anchor, Identity Pose Space, and AnchorSpace. I think I just forgot and did the intuitive (albeit incorrect) thing.